app/src/main/java/com/globant/practice/presentation/view/adapter/SubscriberAdapter.java

```diff
@@ -120,6 +122,8 @@ public SubscriberViewHolder(View itemView) {
         public void blindSubscriberList(final User user) {
             nickname.setText(user.getLogin());
             this.user = user;
+            Glide.with(context).load(user.getAvatarUrl()).centerCrop()
```

Comment:
is only safe for the string null, not for the context or the imageview null, I will make a commit to prevent that crash, thanks.